cinema/serializers.py

```diff
@@ -1 +1,76 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    Genre,
+    Actor,
+    CinemaHall,
+    Movie,
+    MovieSession,
+)
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ("id", "name")
+        read_only_fields = ("id",)
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ("id", "first_name", "last_name", "full_name")
```

Comment:
The `full_name` field in `ActorSerializer` should be marked as read-only since it is a property and not a model field. Consider adding `full_name` to the `read_only_fields` tuple.